app/views/bgeigie_imports/_status.html.erb

```diff
@@ -11,13 +11,20 @@
     <%= @bgeigie_import.status.humanize %>
   </span>
 <%- end -%>
-<%- if @bgeigie_import.rejected == true -%>
-  <span class="label label-warning">
-    Rejected by: <%= @bgeigie_import.rejected_by %>
-  </span><span style="color: #202020;font-size: 13px;font-family: proxima-nova, sans-serif;">
-  <b>comment:</b>
-  <%= @bgeigie_import.comment if @bgeigie_import.comment.present? %>
-</span>
+<%- if user_signed_in? && (current_user.moderator? || current_user == @bgeigie_import.user)-%>
```

Comment:
Maybe not for this PR, but would be good to turn this pattern into a helper function. May something like `approval_participant?(current_user, @bgeigie_import)`

Feel free to implement it if you like that idea.